### ISSUE TITLE
Hi, using nnunet encoder for classification

### ISSUE THREAD

**Issue by provbs** [MIC-DKFZ/nnUNet#1521]
Hi, I was trying to use the nnunet encoder for classification,

adding a simple classifier at the bottleneck right after encoder.

```
for d in range(len(self.conv_blocks_context) - 1):
            x = self.conv_blocks_context[d](x)
            skips.append(x)
            if not self.convolutional_pooling:
                x = self.td[d](x)

        x = self.conv_blocks_context[-1](x)

        aux1 = self.aux1(x)
```

where aux1 is the classifier

I returned the aux1 value to loss and calculated the loss with crossentorpyloss in pytorch.

I gave the model the right target,

the nnunet_train goes well with no error,

but the classifier loss is not decreasing, which seems like it is not training at all..

It starts with 1.8 ish loss and still at 1.6 ish after 3000 epochs..

Could you give me any advise to make it right?

Removing decoder part and then adding that classifier was all I did..
